src/illumidesk/illumidesk/apis/nbgrader_service.py

```diff
@@ -51,7 +48,7 @@ class NbGraderServiceHelper:
       database_name: the database name
     """
 
-    def __init__(self, course_id: str, check_database_exists: bool = False):
```

Comment:
So what I'm understanding here is that all courses will be in one database instead of a database-specific for each org_course. This assumes that the database exists by default.

Reply:
right, the DB will be common and will be maintained by the migrations in admin-dashboard code base

Reply:
we may need to add this back in case the grader does not exist

Reply:
we may need to add this back in case the grader does not exist